PocketKit/Sources/Analytics/AppEvents/Search.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  Search.swift
+//  
+//
+//  Created by Daniel Brooks on 2/13/23.
+//
+
+import Foundation
+
+public extension Events {
+    struct Search {}
```

Comment:
This file and setup should mirror android. See https://github.com/Pocket/pocket-ios/blob/mirror-android/PocketKit/Sources/Analytics/AppEvents/Home.swift as an example

Reply:
Androids version is at https://github.com/Pocket/Android/blob/feature-analytics/analytics/src/main/java/com/pocket/analytics/appevents/HomeEvents.kt